### PR TITLE
Update ripple.com » xrpl.org links

### PR DESCRIPTION
**Pull request by WietseWind**
### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [x] Documentation Updates
- [ ] Release

### CODE REVIEW

**Review by carlhua — APPROVED** [review]
LGTM, thanks for the update @WietseWind

File: README.md [hunk 1]
```diff
@@ -53,9 +53,9 @@ git-subtree. See those directories' README files for more details.
...
 
-* [XRP Ledger Dev Portal](https://developers.ripple.com/)
-* [XRP News](https://ripple.com/category/xrp/)
-* [Setup and Installation](https://developers.ripple.com/install-rippled.html)
+* [XRP Ledger Dev Portal](https://xrpl.org/)
+* [XRP News](https://blog.ripplex.io/xrp/)
+* [Setup and Installation](https://xrpl.org/install-rippled.html)
 * [Doxygen](https://ripple.github.io/rippled)
 
 To learn about how Ripple is transforming global payments, visit
```
Comment (nbougalis) [inline README.md:61]: While you're here, please remove this unnecessary advertisment.

File: README.md [hunk 2]
```diff
@@ -53,9 +53,9 @@ git-subtree. See those directories' README files for more details.
 
 ## See Also
 
-* [XRP Ledger Dev Portal](https://developers.ripple.com/)
-* [XRP News](https://ripple.com/category/xrp/)
-* [Setup and Installation](https://developers.ripple.com/install-rippled.html)
+* [XRP Ledger Dev Portal](https://xrpl.org/)
+* [XRP News](https://blog.ripplex.io/xrp/)
```
Comment (nbougalis) [inline README.md:57]: I know this was mostly a mechanical rename, but this URL is dead and also, it's just RippleX's blog. I don't know if it needs to be in here, but even if it is, it should not be named "XRP News". Honestly, I'd rather just remove it.

**Review by nbougalis — CHANGES_REQUESTED** [review]
Thanks for doing this; the **README** needs a lot of love, but this is a good first step. Left a couple of comments that should be addressed before this is merged.

File: README.md (hunk 2 above)
Comment (intelliot) [reply README.md:57]: +1 for removing it

**Review by mDuo13 — APPROVED** [review]
I applied the changes @nbougalis suggested and added a couple more relevant xrpl.org links. :+1: now.

**Review by intelliot — APPROVED** [review]
Awesome

**Review by nbougalis — APPROVED** [review]

**Pull request closed without merging**